[DOCS] Manage Endpoint - UX Updates (#1010)
elastic/security-docs: docs/management/admin/admin-pg-ov.asciidoc

* *Policy:* The name of the associated policy when the agent was installed. Click the link to view the Integration policy page.

* *Policy Status:* Lists whether the policy application was a success or failure. Click the link to view response details in a flyout.

* *OS:* The associated operating system.

* *IP address:* All IP addresses associated with the hostname.

* *Version:* The {stack} version currently running.

* *Last active:* A date and timestamp of the last time the agent was active.

* *Actions:* Select the context menu (*...*) to do the following:

** *Isolate host:* <<host-isolation-ov, Isolate the host>> from your network, blocking communication until the host is released.

** *View host details:* View host details on the *Hosts* page in the {security-app}.

** *View agent policy:* View the policy in {fleet}.

** *View agent details:* View agent details and activity logs in {fleet}.

** *Reassign agent policy:* Change the {fleet-guide}/agent-policy.html#apply-a-policy[agent policy] assigned to the host in {fleet}.

[role="screenshot"]
image::images/endpoints-pg.png[Admin page]

*Endpoint details*

> Click any link in the *Endpoint* column to display host details in a flyout. You can also use the *Take Action* menu button to perform the same actions as those listed in the Actions context menu (such as isolating the host, viewing host details, viewing and reassigning the agent policy, and so on).

Reviewer: ```suggestion
Click any link in the *Endpoint* column to display host details in a flyout. You can also use the *Take Action* menu button to perform the same actions as those listed in the Actions context menu, such as isolating the host, viewing host details, and viewing or reassigning the agent policy. 
```

[role="screenshot"]
image::images/filter-endpoints.png[]

> NOTE: The time picker on the right side of the page allows you to set a time interval to automatically refresh the Endpoints list — for example, if new endpoints were added or deleted.

Reviewer 2: Small nite - I typically refer to the time picker feature as the "date and time picker" in my docs but I've also seen it labeled as the "date picker" (see the "Host Key Performance Indicators (KPIs) charts" section in this topic) in our docs so might be worth it to come up with a single term for this one. :)

Author: Good point! I like "date and time picker" since it's more accurate. And yes, a good term to standardize.
